### PR TITLE
Added some magic posh to clear run history

### PR DESCRIPTION
**Pull request by Akshay-Rohatgi**

### CODE REVIEW

File: src/release_windows.go [hunk 1]
```diff
@@ -152,4 +152,17 @@ func cleanUp() {
 	if verboseEnabled {
 		warnPrint("Done with automatic cleanup! You need to remove aeacus.exe manually. The only things you need in the C:\\aeacus directory is phocus, scoring.dat, TeamID.txt, and the assets directory.")
 	}
+	if verboseEnabled {
+		warnPrint("clearing run history")
+	}
```
Comment (aaditgupta21) [inline src/release_windows.go:157]: do we still wanna do verboseEnabled here?

Comment (safinsingh) [reply src/release_windows.go:157]: It should be until #58 is merged

Comment (aaditgupta21) [reply src/release_windows.go:157]: oh your right

Comment (Akshay-Rohatgi) [reply src/release_windows.go:157]: well shanes thing hasnt been integrated into aeacus yet, i can wait for shanes pr to be merged and then change mine

**Review by Altoid0 — APPROVED** [review]

**Pull request merged**